### PR TITLE
make provider a configuration

### PR DESCRIPTION
**Pull request by tevino**
For those who have no access to [code](http://code.dapps.douban.com/), they may have this two lines in their `.codecli.conf`:
```
[provider]
    name = github
```

### CODE REVIEW

**Comment by tevino** [comment]
@hongqn

File: codecli/utils.py [hunk 1]
```diff
@@ -157,6 +157,12 @@ def get_code_username():
     return user_name
 
 
+def get_default_provider():
+    provider = get_config('provider.name')
+    print provider
```
Comment (hongqn) [inline codecli/utils.py]: no print in library please.

Comment (tevino) [inline codecli/utils.py]: Sorry, my fault.

**Comment by tevino** [comment]
@hongqn :joy:

**Comment by hongqn** [comment]
Sorry for the late merging.  I have not checked my email for too long... :bow:

**Pull request merged**